socorro/cron/jobs/bugzilla.py

```diff
@@ -106,46 +109,56 @@ class BugzillaCronApp(BaseCronApp):
     )
 
     def run(self):
-        # if this is non-zero, we use it.
+        # Figure out how far back we want to ask Bugzilla about
         if self.config.days_into_past:
-            last_run = (
-                utc_now() -
-                datetime.timedelta(days=self.config.days_into_past)
-            )
+            last_run = utc_now() - datetime.timedelta(days=self.config.days_into_past)
+
         else:
             try:
-                # KeyError if it's never run successfully
+                # KeyError if it has never run successfully
                 # TypeError if self.job_information is None
                 last_run = self.job_information['last_success']
             except (KeyError, TypeError):
-                # basically, the "virgin run" of this job
+                # This job has never been run
                 last_run = utc_now()
 
         # bugzilla runs on PST, so we need to communicate in its time zone
         PST = tz.gettz('PST8PDT')
         last_run_formatted = last_run.astimezone(PST).strftime('%Y-%m-%d')
+
+        # Fetch recent relevant changes and iterate over them updating our
+        # data set
         for bug_id, signature_set in self._iterator(last_run_formatted):
-            # each run of this loop is a transaction
             self.database_transaction_executor(
-                self.inner_transaction,
+                self.update_bug_data,
                 bug_id,
                 signature_set
             )
 
-    def inner_transaction(self, connection, bug_id, signature_set):
-        self.config.logger.debug("bug %s: %s", bug_id, signature_set)
+    def update_bug_data(self, connection, bug_id, signature_set):
+        self.update_bug_data_old_table(connection, bug_id, signature_set)
+        self.update_bug_data_new_table(connection, bug_id, signature_set)
```

Comment:
The way I did this involves a lot of duplicated stuff, but it'll be much easier to remove in a couple of days.